src/components/TimelineOnboarding.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import { ref, computed, watch } from 'vue';
+import { useI18n } from '@/composables/useI18n';
+import { useOnboardingChecklist } from '@/composables/useOnboardingChecklist';
+import { lsSet, lsGet } from '@/helpers/utils';
+
+const { t } = useI18n();
+
+const { onboardingChecklist } = useOnboardingChecklist();
+
+const showOnboarding = ref(true);
+
+const checkListCompleted = computed(() => {
+  let completed = true;
+
+  onboardingChecklist.value.forEach(checkListItem => {
+    if (!checkListItem.checked) {
+      completed = false;
+      return;
+    }
+  });
+
+  return completed;
+});
+
+if (lsGet('timelineOnboarding') == 'hidden') {
+  showOnboarding.value = false;
+}
+
+watch(checkListCompleted, completed => {
+  if (completed) lsSet('timelineOnboarding', 'hidden');
```

Comment:
Hi @ManuBernard thanks for your contribution!

You can import `useStorage` which would simplify the storage logic of this component a lot. https://vueuse.org/core/useStorage/

Reply:
Also the storage key should probably include a substring of the connected address, incase users are sharing the same device. e.g how we do it in other places `web3Account.value.slice(0, 8)`

Reply:
Hi @samuveth ,  thanks for your feedbacks. 
I have added the account substring and used useStorage, it changes slightly the behavior so I added a delay before it closes the onboarding panel. 

I hope it is fine like this!